src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -295,23 +295,20 @@ def _send_verification_code_to_user(
     )  # Validates Fidesops is currently configured to send emails
     verification_code: str = generate_id_verification_code()
     privacy_request.cache_identity_verification_code(verification_code)
-    dispatch_email_task.apply_async(
-        queue=EMAIL_QUEUE_NAME,
-        kwargs={
-            "email_meta": FidesopsEmail(
-                action_type=EmailActionType.SUBJECT_IDENTITY_VERIFICATION,
-                body_params=SubjectIdentityVerificationBodyParams(
-                    verification_code=verification_code,
-                    verification_code_ttl_seconds=config.redis.identity_verification_code_ttl_seconds,
-                ),
-            ).dict(),
-            "to_email": email,
-        },
+    # synchronous call for now since failure to send verification code is fatal to request
```

Comment:
we'll update all remaining synchronous calls to use Celery in this ticket- https://github.com/ethyca/fidesops/issues/1357